units/systemd-timesyncd.service.in

```diff
@@ -37,7 +38,8 @@ RestrictNamespaces=yes
 RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
 SystemCallFilter=~@cpu-emulation @debug @keyring @module @mount @obsolete @raw-io @reboot @swap
 SystemCallArchitectures=native
-ReadWritePaths=/var/lib/systemd
+StateDirectory=systemd/timesync
+User=systemd-timesync
```

Comment:
Minor personal preference: Could you move the User option near the top of the Service section and add Group option even if it is implied by the User option?